pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -60,7 +60,7 @@ var (
 	// higher-level, telemetry-friendly metrics
 	vmiCountDesc = prometheus.NewDesc(
 		"kubevirt_vmi_phase_count",
-		"VMI phase.",
+		"Sum of VMIs per phase and node. `phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]",
```

Comment:
nit: to make this refactoring safe, you might consider using an array holding the constant values and using the string representation

Reply:
Just committed the changes, wdyt?